hackathon_site/dashboard/frontend/src/pages/IncidentForm/IncidentForm.tsx

```diff
@@ -0,0 +1,255 @@
+                                                            <RadioGroup
+                                                                {...field}
+                                                                value={field.value}
+                                                                onChange={
+                                                                    field.onChange
+                                                                }
+                                                                data-testid="radio-state"
+                                                                row
+                                                            >
+                                                                <FormControlLabel
+                                                                    value="broken"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Broken"
+                                                                />
+                                                                <FormControlLabel
+                                                                    value="lost"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Lost"
+                                                                />
+                                                                <FormControlLabel
+                                                                    value="other"
+                                                                    control={
+                                                                        <Radio color="primary" />
+                                                                    }
+                                                                    label="Other"
+                                                                />
+                                                            </RadioGroup>
+                                                        </>
+                                                    )}
+                                                </Field>
+                                                <FormHelperText>
+                                                    {errors?.state}
+                                                </FormHelperText>
+                                            </FormControl>
+                                            <br></br>
```

Comment:
Using `<br>` tags to create space between elements is not good practice. 

For virtually anything related to layout, you should use CSS instead. So if you want to create more space between blocks of text or other things, instead of using the <br> tag, you should use CSS margin or padding.